src/extension.ts

```diff
@@ -357,6 +356,11 @@ export async function activate(context: vscode.ExtensionContext) {
 
     const debugProvider = new IdfDebugConfigurationProvider();
     context.subscriptions.push(vscode.debug.registerDebugConfigurationProvider("cppdbg", debugProvider));
+
+    registerIDFCommand("espIdf.getProjectName", () => {
+        return getProjectName(workspaceRoot.fsPath);
```

Comment:
maybe we need to await here, since this returns a promise and not the project name, its always a good practice to return concrete value or display error to the user here, when the command is executed